napari/utils/events/evented_model.py

```diff
@@ -198,20 +198,33 @@ def __init__(self, **kwargs):
 
         self._events.source = self
         # add event emitters for each field which is mutable
-        event_names = [
-            name
-            for name, field in self.__fields__.items()
-            if field.field_info.allow_mutation
-        ]
-        event_names.extend(self.__property_setters__)
-        self._events.add(**dict.fromkeys(event_names))
+        field_events = []
+        for name, field in self.__fields__.items():
+            value = getattr(self, name)
+            if hasattr(value, 'events'):
+                field_events.append(name)
```

Comment:
this is the cause for the broken test... you're no longer adding adding events for top level objects, hence the error ` AttributeError: 'EmitterGroup' object has no attribute 'octree'`

Reply:
oef! Thanks, lost this among the rest!